test/extended/cluster/cl.go

```diff
@@ -202,6 +208,19 @@ func newProject(nsName string) *projectapi.Project {
 	}
 }
 
+func labelNamespace(c kclientset.Interface, ns string, pairs map[string]string) (*v1.Namespace, error) {
```

Comment:
Please move this into `utils.go`, you can see there are a host of dependencies being imported here (in this commit) that for the most part already are in utils.

The function name is unclear to me... Are we trying to `SetNamespaceLabels` or `UpdateNamespaceLabels`?

Reply:
Done.
Moved and renamed to SetNamespaceLabels